pkg/splunk/client/awss3client.go

```diff
@@ -77,6 +80,9 @@ func InitAWSClientSession(region, accessKeyID, secretAccessKey string) SplunkAWS
 		return nil
 	}
 
+	// TODO: Fix this and store it in the SplunkAWSS3Client
+	gSess = sess
```

Comment:
@gaurav-splunk I may need you help with merging this into the correct interface since its different (`s3manager`) than the `s3` object used by the list APIs